syntax = "proto3";

message Number {
    float value = 1;
}

message HelloRequest {
  string greeting = 1;
}

service Calculator {
    rpc SquareRoot(Number) returns (Number) {}
    rpc StrName(HelloRequest) returns (HelloRequest) {}
}